src/debug/createdump/dumpwriter.cpp

```diff
@@ -61,7 +61,7 @@ DumpWriter::Release()
 bool
 DumpWriter::OpenDump(const char* dumpFileName)
 {
-    m_fd = open(dumpFileName, O_WRONLY|O_CREAT|O_TRUNC, 0664);
+    m_fd = open(dumpFileName, O_WRONLY|O_CREAT|O_TRUNC, S_IWUSR | S_IRUSR);
```

Comment:
@hoyosjs does this need to get ported to main?

Reply:
Yes, I have it in a branch and was waiting for this PR to make it out today. Will put up the main one now.